TFT/src/User/API/UI/TouchProcess.c

```diff
@@ -144,28 +146,15 @@ uint16_t Key_value(uint8_t total_rect, const GUI_RECT *menuRect)
 
 void loopTouchScreen(void)  // Handle in interrupt
 {
-  static uint8_t touch;
-  if (!XPT2046_Read_Pen())
-  {
-    if (touch >= 20)  // 20ms
-    {
-      touchScreenIsPress = true;
-    }
-    else
-    {
-      touch++;
-    }
-  }
+  if (XPT2046_Read_Pen() == LOW)
+    touchCountdown -= !!touchCountdown;
```

Comment:
This method is not acceptable as it depends on the number of times this API is called and is limited by the entire task cycle of the large loop.
It is easy for the filtering time to be too short, not performing the filtering function it should have, or the filtering time to be too long, resulting in slow touch.